calendar_sk.py

```diff
@@ -0,0 +1,196 @@
+     'Nikola, Nikolaj', 'Ružena', 'Nora'],
+
+    # september
+    ['Drahoslava', 'Linda, Rebeka', 'Belo', 'Rozália', 'Regina', 'Alica',
+     'Marianna', 'Miriama', 'Martina', 'Oleg', 'Bystrík', 'Mária, Marlena',
+     'Ctibor', 'Ľudomil', 'Jolana', 'Ľudmila', 'Olympia', 'Eugénia',
+     'Konštantín', 'Ľuboslav, Ľuboslava', 'Matúš', 'Móric', 'Zdenka',
+     'Ľuboš, Ľubor', 'Vladislav, Vladislava', 'Edita', 'Cyprián', 'Václav',
+     'Michal, Michaela', 'Jarolím'],
+
+    # october
+    ['Arnold', 'Levoslav', 'Stela', 'František', 'Viera', 'Natália', 'Eliška',
+     'Brigita', 'Dionýz', 'Slavomíra', 'Valentína', 'Maximilián', 'Koloman',
+     'Boris', 'Terézia', 'Vladimíra', 'Hedviga', 'Lukáš', 'Kristián',
+     'Vendelín', 'Uršuľa', 'Sergej', 'Alojzia', 'Kvetoslava', 'Aurel',
+     'Demeter', 'Sabína', 'Dobromila', 'Klára', 'Šimon, Simona', 'Aurélia'],
+
+    # november
+    ['Denis, Denisa', None, 'Hubert', 'Karol', 'Imrich', 'Renáta', 'René',
+     'Bohumír', 'Teodor', 'Tibor', 'Martin, Maroš', 'Svätopluk', 'Stanislav',
+     'Irma', 'Leopold', 'Agnesa', 'Klaudia', 'Eugen', 'Alžbeta', 'Félix',
+     'Elvíra', 'Cecília', 'Klement', 'Emília', 'Katarína', 'Kornel', 'Milan',
+     'Henrieta', 'Vratko', 'Ondrej, Andrej'],
+
+    # december
+    ['Edmund', 'Bibiána', 'Oldrich', 'Barbora, Barbara', 'Oto', 'Mikuláš',
+     'Ambróz', 'Marína', 'Izabela', 'Radúz', 'Hilda', 'Otília', 'Lucia',
+     'Branislava, Bronislava', 'Ivica', 'Albína', 'Kornélia', 'Sláva',
+     'Judita', 'Dagmara', 'Bohdan', 'Adela', 'Nadežda', 'Adam, Eva', None,
+     'Štefan', 'Filoména', 'Ivana, Ivona', 'Milada', 'Dávid', 'Silvester'],
+]
+
+
+holiday = {
+    '1': {'1': 'Deň vzniku Slovenskej republiky',
+          '6': 'Zjavenie Pána (Traja králi)'},
+
+    '5': {'1': 'Sviatok práce',
+          '8': 'Deň víťazstva nad fašizmom'},
+
+    '7': {'5': 'Sviatok svätého Cyrila a Metoda'},
+
+    '8': {'29': 'Výročie SNP'},
+
+    '9': {'1': 'Deň Ústavy Slovenskej republiky',
+          '15': 'Sedembolestná Panna Mária'},
+
+    '11': {'1': 'Sviatok všetkých svätých',
+           '17': 'Deň boja za slobodu a demokraciu'},
+
+    '12': {'24': 'Štedrý deň',
+           '25': 'Prvý sviatok vianočný',
+           '26': 'Druhý sviatok vianočný'}
+}
+
+
+INTERNATIONAL_DAY = {
+    '1': {'27': 'Medzinárodný deň pamiatky obetí holokaustu '},
+
+    '2': {'4': 'Svetový deň proti rakovine ',
+          '8': 'Deň pre bezpečnejší internet ',
+          '14': 'Deň zaľúbených',
+          '15': 'Deň osamelých'},
+
+    '3': {'8': 'Medzinárodný deň žien',
+          '28': 'Deň učiteľov'},
+
+    '4': {'11': 'Deň narcisov – deň boja proti rakovine',
+          '22': 'Deň zeme'},
+
+    '5': {'10': 'Deň matiek'},
+
+    '6': {'1': 'Medzinárodný deň detí',
+          '21': 'Deň otcov'},
+
+    '7': {'8': 'Deň videohier',
+          '26': 'Deň rodičov'},
+
+    '8': {'13': 'Medzinárodný deň ľavákov'},
+
+    '9': {'13': 'Medzinárodný deň programátorov',
```

Comment:
@Adman actually, as far as I know, this day can be on 12th of September in case of leap year (see [wikipedia article](https://en.wikipedia.org/wiki/Programmers%27_Day)).